Common/Securities/SecurityTransactionManager.cs

```diff
@@ -353,5 +356,32 @@ private static bool Completed(Order order)
         {
             return order.Status == OrderStatus.Filled || order.Status == OrderStatus.PartiallyFilled || order.Status == OrderStatus.Invalid || order.Status == OrderStatus.Canceled;
         }
+
+        /// <summary>
+        /// Returns true when order quantity close or reduce long position
+        /// </summary>
+        /// <param name="order">Order we're checking</param>
+        /// <param name="security">Security that the order tries to close or reduce</param>
+        /// <param name="value">Remaining quantity to close long position</param>
+        /// <returns>True if order close or reduce long position</returns>
+        private bool TryGetRemainingQuantity(Order order, Security security, out int value)
+        {
+            value = 0;
+            if (order.Direction != OrderDirection.Sell) return false;
```

Comment:
Buying can be used to close a short position. I don't think this can be based on the order direction.